apps/filecoin-site/src/app/_components/SectionSubContent.tsx

```diff
@@ -23,12 +23,12 @@ export function SectionSubContent({
       <div className="max-w-2xl">
         <Heading
           tag={headingTag}
-          className="text-xl font-medium tracking-tighter md:text-2xl"
+          className="text-xl font-medium md:text-3xl/10"
         >
           {title}
         </Heading>
         {description && (
-          <p className="pt-6 text-lg/7 text-pretty text-[var(--color-paragraph-text)]">
+          <p className="pt-6 text-xl/7 text-pretty text-[var(--color-paragraph-text)]">
```

Comment:
Why are we removing text-lg/7 on smaller screens? Is it because the design doesn’t include it? Just want to make sure. Filipa might not have updated that part yet, so we should align before changing.

Reply:
Are the `p` styles in `apps/filecoin-site/src/app/_components/SectionContent.tsx` related? Flagging just in case we end up making changes here.